apps/schools/api_views/boundary.py

```diff
@@ -41,7 +41,7 @@ class ParliamentList(KLPListAPIView):
     bbox_filter_field = 'coord'
 
     def get_queryset(self):
-        qset = Parliament.objects.all()
+        qset = Parliament.objects.none()
 
```

Comment:
We don't have parliament data for Odisha yet. This reflects the Electoral boundaries at `http://odisha.ilp.org.in/reports/search`.